test/helper.py

```diff
@@ -16,26 +21,67 @@ def evaluate(str_expr: str):
 def check_evaluation(
     str_expr: str,
     str_expected: str,
-    message="",
-    to_string_expr=True,
-    to_string_expected=True,
-    to_python_expected=False,
+    message: str = "",
+    hold_expected: bool = False,
+    to_string_expr: bool = True,
+    to_string_expected: bool = True,
+    to_python_expected: bool = False,
+    msgs=None,
```

Comment:
Adding `msgs` s a much welcome and needed improvement. 

It would be nice to change the name to something like `messages_expected` make it more distinct from `message`. And maybe `message` should be something like `test_failure_message`.